Adyen/Core/Payment Methods/StoredPayByBankUSPaymentMethod.swift

```diff
@@ -6,8 +6,8 @@
 
 import Foundation
 
-/// Stored PayByBank payment.
-public struct StoredPayByBankPlaidPaymentMethod: StoredPaymentMethod {
+/// Stored PayByBank US payment.
+public struct StoredPayByBankUSPaymentMethod: StoredPaymentMethod {
```

Comment:
⚠️ This is a fully breaking change, can we keep struct and maybe create a typealias?

Reply:
I found `StoredPayByBankPlaidPaymentMethod` was added in the base branch. Let's rethink which branch to target when we run api diff? Options:
- develop (then we might have all long-living branch changes)
- latest release tag (then every time we have all development changes combined)

Or we can switch off api diff for all PRs not targeting develop and deal with all changes when we merge long-living branch into develop. The only concern I have is that usually it's a common sentiment to merge as fast as possible because "everything was reviewed", everyone has seen the code multiple times and naturally you just want to merge it asap.